src/analytics/event.ts

```diff
@@ -137,6 +137,10 @@ export const event = {
    * Called when the dapps hits the rate limit per second or minute.
    */
   dappProviderRateLimit: 'dapp.provider.rate_limit',
+  /**
+   * Tracks when the extension popup is opened via keyboard shortcut (Alt+Shift+R)
+   */
+  extensionOpenViaShortcut: 'extension.open.shortcut',
```

Comment:
Nitpicky, but might rename to `browserShortcutTriggered` with a `action` type (i.e. `launch`) to align with the other `keyboardShortcutTriggered` and `keyboardNavigationTriggered` events. We may add other actions in the future

Reply:
lemme know if that works.